SmartDeviceLink/public/SDLVideoStreamingCapability.h

```diff
@@ -30,7 +31,22 @@ NS_ASSUME_NONNULL_BEGIN
  @param scale The scaling factor the app should use to change the size of the projecting view
  @return A SDLVideoStreamingCapability object
  */
-- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale;
+- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale __deprecated_msg("Use initWithPreferredResolution:maxBitrate:supportedFormats:hapticDataSupported:diagonalScreenSize:pixelPerInch:scale:preferredFPS instead");
+
+/**
+ Convenience init for creating a video streaming capability with all parameters.
+
+ @param preferredResolution The preferred resolution of a video stream for decoding and rendering on HMI
+ @param maxBitrate The maximum bitrate of video stream that is supported, in kbps
+ @param supportedFormats Detailed information on each format supported by this system, in its preferred order
+ @param hapticDataSupported True if the system can utilize the haptic spatial data from the source being streamed
+ @param diagonalScreenSize The diagonal screen size in inches
+ @param pixelPerInch The diagonal resolution in pixels divided by the diagonal screen size in inches
+ @param scale The scaling factor the app should use to change the size of the projecting view
+ @param preferredFPS The preferred frame rate per second of the head unit
+ @return A SDLVideoStreamingCapability object
+ */
+- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale preferredFPS:(uint32_t)preferredFPS;
```

Comment:
```suggestion
/**
 * @param preferredResolution - preferredResolution
 * @param maxBitrate - maxBitrate
 * @param supportedFormats - supportedFormats
 * @param hapticSpatialDataSupported - hapticSpatialDataSupported
 * @param diagonalScreenSize - diagonalScreenSize
 * @param pixelPerInch - pixelPerInch
 * @param scale - scale
 * @param preferredFPS - preferredFPS
 * @return A SDLVideoStreamingCapability object
 */
- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(nullable NSNumber<SDLUInt> *)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticSpatialDataSupported:(nullable NSNumber<SDLBool> *)hapticSpatialDataSupported diagonalScreenSize:(nullable NSNumber<SDLFloat> *)diagonalScreenSize pixelPerInch:(nullable NSNumber<SDLFloat> *)pixelPerInch scale:(nullable NSNumber<SDLFloat> *)scale preferredFPS:(nullable NSNumber<SDLUInt> *)preferredFPS;
```

Reply:
Updated SDLVideoStreamingCapability.h based on the generated code as you suggested.